audio.py

```diff
@@ -1,11 +1,42 @@
 from pydub import AudioSegment
+import urllib2
+import string
 
 
-def load_mp3(mp3_file_name):
+def load_mp3(mp3_file_name, file_type):
+
+    """
+    This check if is a url and donwload the file
+    in files directory with podcast.mp3 filename.
+    """
+    if mp3_file_name.startswith('https://') or mp3_file_name.startswith('http://'):
+        """
+        This check if is a dropbox URL and set paramater
+        that allows download the file.
+        """
```

Comment:
Creo que no se usan docstrings para comentarios comunes, sólo para documentar la función.

Reply:
Instead of this docstring, I would use this:
```python
   # Check the URL is a Dropbox one, then set the download parameter
```

Reply:
Done!